test/test_art.py

```diff
@@ -157,13 +165,21 @@ class CombinedTest(FetchImageHelper, UseThePlugin):
     CAA_URL = 'http://coverartarchive.org/release/{0}/front' \
               .format(MBID)
 
+    AMAZON_URL_HTTPS = 'http://images.amazon.com/images/P/{0}.01.LZZZZZZZ.jpg' \
+                 .format(ASIN)
+    AAO_URL_HTTPS = 'http://www.albumart.org/index_detail.php?asin={0}' \
+                 .format(ASIN)
+    CAA_URL_HTTPS = 'http://coverartarchive.org/release/{0}/front' \
+                 .format(MBID)
```

Comment:
Is there any way we can avoid copying and pasting here? For example, we could put most of the URLs (starting with `://`) into a variable and then concatenate on the `https` or `http` part separately.